New package: OpenChemistry.Avogadro2 version 1.91.0

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Related to #156209, but failed to solve it.

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/156213)

Dragon1573 committed:
New package: OpenChemistry.Avogadro2 version 1.91.0

Signed-off-by: 是蓝兔哟～ <49941141+Dragon1573@users.noreply.github.com>

stephengillie commented:
Automatic Validation ended with:

> Executable C:\Program Files\Avogadro2\bin\avogadro2.exe returned exit code: -1073740771

(Automated response - build 882.)

Trenly commented on manifests/o/OpenChemistry/Avogadro2/1.91.0/OpenChemistry.Avogadro2.locale.en-US.yaml:
```diff
@@ -0,0 +1,20 @@
+PackageLocale: en-US
+Publisher: http://openchemistry.org/
+PublisherUrl: https://www.openchemistry.org/
+PackageName: Avogadro 2
```
```suggestion
PackageName: Avogadro2
```
This should match Control Panel

Trenly reviewed (changes requested):
Also worth noting that `1.99.0` is the latest version

stephengillie commented:
Manual Validation ended with:
> 
> Faulting application name: avogadro2.exe, version: 0.0.0.0, time stamp: 0x5b7d7773 
> Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000 Exception code: 0xc000041d Fault offset: 0x0000000000000000 
> Faulting process id: 0x12b8 
> Faulting application start time: 0x01dab61695f10148 
> Faulting application path: C:\Program Files\Avogadro2\bin\avogadro2.exe 
> Faulting module path: unknown Report Id: 956accb1-e848-4833-a329-f772e791275e 
> Faulting package full name:  
> Faulting package-relative application ID:   Fault bucket 1875445789161788192, type 4 Event Name: APPCRASH Response: Not available Cab Id: 0  Problem signature: P1: avogadro2.exe P2: 0.0.0.0 P3: 5b7d7773 P4: StackHash_e372 P5: 0.0.0.0 P6: 00000000 P7: c000041d P8: PCH_84_FROM_unknown+0x0000000000000000 P9:  P10:   Attached files: \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERC303.tmp.dmp \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERC352.tmp.WERInternalMetadata.xml \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERC353.tmp.xml \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERC363.tmp.csv \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERC373.tmp.txt  These files may be available here: \\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_avogadro2.exe_edba6658a4290d49620c1424c999da124691_72ae8156_85359718-1927-4e7c-8a11-3fed3d09db1d  Analysis symbol:  Rechecking for solution: 0 Report Id: 956accb1-e848-4833-a329-f772e791275e Report Status: 268435456 Hashed bucket: 51f4df3fad9ab840ea06ec023a7d9320 Cab Guid: 0

(Automated response - build 883.)

Dragon1573 committed:
Apply suggestions from code review

- Change package name in manifest
- It should match that in Control Panel exactly

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

Dragon1573 commented:
See https://github.com/microsoft/winget-pkgs/pull/156213#pullrequestreview-2095072487 for details.
